apps/desktop/src/database/artifacts/runtime/seed.sql

```
@@ -125,6 +125,12 @@ INSERT INTO built_in_tools (
 SELECT 'web_fetch', 'WebFetch', '抓取网页并提取易读文本', 1, 'low', NULL
 WHERE NOT EXISTS (SELECT 1 FROM built_in_tools WHERE tool_id = 'web_fetch');
 
+INSERT INTO built_in_tools (
+    tool_id, display_name, description, enabled, risk_level, config_json
+)
+SELECT 'web_browse', 'WebBrowse', '浏览交互式网页，支持 JavaScript 渲染和页面交互', 1, 'low', NULL
+WHERE NOT EXISTS (SELECT 1 FROM built_in_tools WHERE tool_id = 'web_browse');
```

SQLFluff (4.2.1)

[error] The 'WHERE' keyword should always start a new line.

(LT14)